### DISCUSSION TITLE
Filter org repos on mobile

### DISCUSSION THREAD

**Discussion by toledompm** [community/community#3309] [Mobile] [answered]
Recently I went into an organization page in order to find a repository, and ended up having to scroll through a list with all 200 of them. A search bar in the org page would be nice :)

**Comment by Learning2Bcoder** [comment]
> Recently I went into an organization page in order to find a repository, and ended up having to scroll through a list with all 200 of them. A search bar in the org page would be nice :)

You can if you go to the select files buttons in the bottom right hand corner of the screen

**Comment by candyho** [comment, marked as answer]
Hey there, thanks for your feedback. We hear you. Stay tuned for more improvements on this front.

**Comment by kroekchai1** [comment]
https://team-1596609309283.atlassian.net/

**Reply by kroekchai1** [reply]
github.kroekchaicup@trueinternet.co.th

**Reply by kroekchai1** [reply]
https://team-1596609309283.atlassian.net/

**Comment by c8llz** [comment]
> Hey there, thanks for your feedback. We hear you. Stay tuned for more improvements on this front.
